jcli/src/jcli_app/rewards/voters.rs

```diff
@@ -0,0 +1,114 @@
+use super::Error;
+use crate::jcli_app::block::Common;
+
+use structopt::StructOpt;
+
+use chain_addr::{Discrimination, Kind};
+use chain_impl_mockchain::vote::CommitteeId;
+use jormungandr_lib::interfaces::{Address, Block0Configuration, Initial};
+use std::collections::{HashMap, HashSet};
+use std::ops::{Div, Mul};
+
+#[derive(StructOpt)]
+#[structopt(rename_all = "kebab-case")]
+pub struct VotersRewards {
+    #[structopt(flatten)]
+    common: Common,
+    /// Reward (in ADA) to be distributed
+    #[structopt(long = "total-rewards")]
+    total_rewards: f64,
```

Comment:
Using floating point numbers for anything money-related is not advised. We should used fixed-point numbers (or just scale everything up to `u64`/`u128`) and do appropriate rounding. And conversions between `u64` and `f64` will result in precision loss.

Reply:
Yeah, maybe it would be better to use Lovelace instead of ADA for this. But what rounding should we use if it is complete fractional?